# DupLoss-2

## **Description**
DupLoss-2 is a program for phylogenomic species tree inference using gene tree parsimony. It takes as input a collection of gene trees and seeks a species tree that best reconciles the input gene trees under a gene duplication and loss reconciliation model. DupLoss-2 can lead to significant improvements in species tree reconstruction accuracy compared to other existing methods on phylogenomic datasets where gene duplication and loss are the primary drivers of gene family evolution. DupLoss-2 is scalable to whole-genome datasets with thousands of gene trees from hundreds of taxa. Further methodological details and experimental results appear in the paper cited below.

This repository includes complete source code, user manual, test data, and precompiled executables for macOS and Linux. In addition, a Python script to automate execution of multiple runs of DupLoss-2 on the same dataset is available in the Executables directory as MultiRunScript.py.

DupLoss-2 can be cited as follows:

<i>DupLoss-2: Improved Phylogenomic Species Tree Inference under Gene Duplication and Loss</i><br>
Rachel Parsons and Mukul S. Bansal<br>
Under review.
